packages/ubuntu_desktop_installer/test/write_changes_to_disk_page_test.dart

```diff
@@ -83,7 +83,10 @@ void main() {
       WidgetTester tester, List<Map<String, dynamic>> storageConfig) async {
     app = MaterialApp(
       supportedLocales: AppLocalizations.supportedLocales,
-      localizationsDelegates: AppLocalizations.localizationsDelegates,
+      localizationsDelegates: [
+        ...AppLocalizations.localizationsDelegates,
+        ...UbuntuLocalizations.localizationsDelegates,
```

Comment:
Just thinking out loud here without having checked the feasibility of my suggestion: is there a way to include `UbuntuLocalizations.localizationsDelegates` in `AppLocalizations.localizationsDelegates` (both here and in the `ubuntu_wsl_setup` package), so that there's no need to copy/paste this snippet all over the place?
Or maybe declare a global static `localizationsDelegates` that can be referenced everywhere?

Reply:
Great idea! :+1:

I can't think of a way to inject extra items into the auto-generated list and extension methods cannot be static, so I went with the second option.

Reply:
LGTM, thanks!